.github/workflows/compose-fastnear-test.yml

```
@@ -0,0 +1,133 @@
+      - main
+  pull_request:
+    paths:
+      - 'indexer/**' # Temporarily util it becomes prime way to obtain NEAR DA blocks.
+
+jobs:
+  run-docker-compose-with-fastnear:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)